packages/firmware_notifier/pubspec.yaml

```diff
@@ -0,0 +1,14 @@
+name: fwupd_notifier
+description: A desktop notification service for fwupd.
+version: 0.0.1
```

Comment:
we could replace the version with `publish_to: 'none'` because it's an internal tool

Reply:
:+1: